sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProtoTest.java

```diff
@@ -63,111 +63,293 @@
 /** Unit tests for {@link org.apache.beam.sdk.io.gcp.bigquery.TableRowToStorageApiProto}. */
 public class TableRowToStorageApiProtoTest {
   // Schemas we test.
-  // The TableRow class has special semantics for fields named "f". To ensure we handel them
+  // The TableRow class has special semantics for fields named "f". To ensure we handle them
   // properly, we test schemas
   // both with and without a field named "f".
   private static final TableSchema BASE_TABLE_SCHEMA =
       new TableSchema()
           .setFields(
               ImmutableList.<TableFieldSchema>builder()
-                  .add(new TableFieldSchema().setType("STRING").setName("stringValue"))
-                  .add(new TableFieldSchema().setType("STRING").setName("f"))
-                  .add(new TableFieldSchema().setType("BYTES").setName("bytesValue"))
-                  .add(new TableFieldSchema().setType("INT64").setName("int64Value"))
-                  .add(new TableFieldSchema().setType("INTEGER").setName("intValue"))
-                  .add(new TableFieldSchema().setType("FLOAT64").setName("float64Value"))
-                  .add(new TableFieldSchema().setType("FLOAT").setName("floatValue"))
-                  .add(new TableFieldSchema().setType("BOOL").setName("boolValue"))
-                  .add(new TableFieldSchema().setType("BOOLEAN").setName("booleanValue"))
-                  .add(new TableFieldSchema().setType("TIMESTAMP").setName("timestampValue"))
-                  .add(new TableFieldSchema().setType("TIME").setName("timeValue"))
-                  .add(new TableFieldSchema().setType("DATETIME").setName("datetimeValue"))
-                  .add(new TableFieldSchema().setType("DATE").setName("dateValue"))
-                  .add(new TableFieldSchema().setType("NUMERIC").setName("numericValue"))
-                  .add(new TableFieldSchema().setType("BIGNUMERIC").setName("bigNumericValue"))
-                  .add(new TableFieldSchema().setType("NUMERIC").setName("numericValue2"))
-                  .add(new TableFieldSchema().setType("BIGNUMERIC").setName("bigNumericValue2"))
+                  .add(
```

Comment:
Should we keep some of these without a mode, in order to test default behavior?

Reply:
reverted all of these changes - no longer needed (they were part of an earlier attempt to test)